dspace-api/src/main/java/org/dspace/ctask/general/MetadataWebServiceDataInfo.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.ctask.general;
+
+import javax.xml.xpath.XPathExpression;
+
+/**
```

Comment:
This class holds an individual mapping from a location in the response document to a desired target metadata field. There may be several mappings  per service call.